batchflow/models/torch/resnet.py

```diff
@@ -115,7 +115,7 @@ class ResNet50(ResNet34):
     @classmethod
     def default_config(cls):
         config = super().default_config()
-        config['body/encoder/blocks/layout'] = 'cna'
+        config['body/encoder/blocks/layout'] = 'cn'
```

Comment:
Why?

Reply:
A bottleneck parameter is set to ```True```, which makes a ResBlock layout equals to ```'cna'``` + config layout + ```'acn'``` (from ResBlock ```__init__``` method).